src/shared/nested-dropdown-list/types.ts

```diff
@@ -0,0 +1,66 @@
+import { L1OptionProps } from "../../input-nested-select";
```

Comment:
feels weird to import circularly but can leave it for now since it doesn't actually break anything

```suggestion
import { L1OptionProps } from "../../input-nested-select/types";
```

Reply:
aww.. thanks spotted that!